sdk/core/Azure.Core/src/Shared/CallerShouldAuditAttribute.cs

```diff
@@ -8,14 +8,14 @@
 namespace Azure.Core
 {
     /// <summary>
-    /// Decorates an operation whose invocation should potentially be audited
+    /// Decorates an operation or all operations for a client whose invocation should potentially be audited
     /// by Azure service implementations.  Auditing could be recommended
     /// because the operation changes critical service state, creates delegated
     /// access to a resource, affects data retention, etc.  It's a best guess
     /// from the service team that the operation should be audited to mitigate
     /// any potential future issues.
     /// </summary>
-    [AttributeUsage(AttributeTargets.Method)]
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
```

Comment:
Curious: what does it mean to audit a class?  Does it mean that all methods on the class will be audited?

Reply:
See my reply to Chris for context: https://github.com/Azure/azure-sdk-for-net/pull/40577#discussion_r1416376832